hist

```diff
@@ -1,29 +1,38 @@
-#!/bin/bash
+#! /usr/bin/env bash
```

Comment:
I think I've never encountered an OS where /usr/bin/env does not exist.
FreeBSD doesn't have /bin/bash, and neither does, for example, NixOS.